project_stage_state/models/project_task.py

```diff
@@ -0,0 +1,13 @@
+# Daniel Reis, 2014
+# GNU Affero General Public License <http://www.gnu.org/licenses/>
+
+from odoo import fields, models
+
+
+class ProjectTask(models.Model):
+    """Added state in the Project Task."""
+
+    _inherit = 'project.task'
+
+    state = fields.Selection(
+        related='stage_id.state', store=True, readonly=True)
```

Comment:
@anandkansagra `related` always `readonly=True` in v12

Reply:
Thanks, Done